dev-util/nvidia-cuda-sdk/Manifest

```diff
@@ -1,3 +1,3 @@
 DIST cuda_6.5.14_linux_32.run 76788277 SHA256 63fa1fe5d013e9944d6c1a3a71768fa4dcb35d6a157c1b06765bb56198c205c5 SHA512 89649ee0f4543b37037b7c8f5ed0b8bd502512afb4d93e3158b3df4786123be05a5288986b5abbfe15c14eb010777d2527b715225bea96a354a898c34e4a2fcc WHIRLPOOL c1094dd84883ef08d91c6eec3634a285496418db1e76b70f449f833dcb01069a620af504b6df7fe2752559688ed1f60f4adda8c97d849d648cba1be0f8196954
 DIST cuda_6.5.14_linux_64.run 972320904 SHA256 f3e527f34f317314fe8fcd8c85f10560729069298c0f73105ba89225db69da48 SHA512 fecca731b86955f5bc7b09d88b02951e3ee5d4b1f1cf5f92658227a029b87cd35d6ac22956a5fa0afe62dcdc57191f1d12bb08c9f7cc05128727cd8455947b1e WHIRLPOOL 0b890771dcca352168fed1b1df9b29504b03f8c29da59cec29914c541283998edf65f5ca6a41362143a2d5d7359822b036894f563c56b39583b1ac4c64e35180
-DIST cuda_7.5.18_linux.run 1195431560 SHA256 245ffb6c1e2b69d323f8ae3a1b63d3384fec3934222e56e9d801ecc0d6e919c5 SHA512 cb933512fde09376b1d6040c72d24c4f91dc44f73322ca87aa7d1f34156c9c87feea41f9a00f1a403cfb89645b5e7a6a5f2bfe4933703ba81f0cf94968d430fd WHIRLPOOL b1a06acba02043fabf496ba683fb984b1c90df2cd71396457f664beb8cb26b2e3074c8568ec6ce0cfe565721986670c7a40351a29a4856a7e03268c14d545247
+DIST cuda_7.5.18_linux.run 1195424633 SHA256 08411d536741075131a1858a68615b8b73c51988e616e83b835e4632eea75eec SHA512 97459dfd124060200f1c43f763e5bce0bea53e35ed4bef72bee6c8ba1c04ecffb8695af11755443513316f91fdb5cb4d35662f7b87bca7e3de85d562d4e49363 WHIRLPOOL 59d186f5e9176db25e5991e8088417ac778953c3ed2f0dcef034968f7ea695c5d4cfea81759ac0f7f1290c3842d9d0339b5b74d2d90ec8b333135aa1b986ca10
```

Comment:
Finally, only the cuda-installer.pl was modified:

~~~
diff -r -x run_files work_new/cuda-installer.pl work_old/cuda-installer.pl
123d122
<     [ [ 'SUSE Linux Enterprise Server 11', '(SP4|PATCHLEVEL\s*=\s*4$)' ], 0, 1, 0 ],
~~~

Furhtermore, the files `run_files/cuda-linux64-rel-7.5.18-19867135.run` differ, but that will be because of the cuda-installer.pl. Thus no revbump necessary.